views/listcourses.php

```diff
@@ -0,0 +1,48 @@
+<?php
+    include_once 'header.php';
+?>
+
+<?php
+    require_once '../models/User.php';
+    require_once '../controllers/ParkingDB.php';
+
+    $hasCourses = false;
+    $courses = array();
+
+    if (is_logged_in()) {
+        $courses = User::getCourses(ParkingDB::getInstance()->getConnection(), $_SESSION['u_id']);
```

Comment:
I think establishing DB connection in the view is not a best practice, consider extracting this logic in a controller.   
(we can leave it like this for now if you think it would slow us down)